src/Calculator/Resources/en-US/Resources.resw

```diff
@@ -4106,7 +4106,167 @@
     <value>Unable to calculate the range for this function.</value>
     <comment>Error displayed when Range is not returned from the analyzer.</comment>
   </data>
-  <data name="equationAnalysisBack.[using:Windows.UI.Xaml.Controls]ToolTipService.ToolTip" xml:space="preserve">
+  <data name="Overflow" xml:space="preserve">
```

Comment:
@grochocki would be nice to have you take a second pass at some of these strings if you have time!

Reply:
does it make sense to give examples of the problem in the error message itself not just in the comment?

Reply:
I don't think we want to provide examples in the error message. It starts to become pretty verbose. Let's start with the simple error messages, and add more detail as necessary based on user feedback.